src/warc2zim/utils.py

```diff
@@ -16,6 +16,16 @@
     re.ASCII,
 )
 
+ENCODING_ALIASES = {}
+
+
+def set_encoding_aliases(aliases: dict[str, str]):
+    """Set the encoding aliases to use to decode"""
+    if not aliases:
+        return
```

Comment:
This contradicts the implied meaning from the function name. If I `set_encoding_aliases({})` I expect all aliases cleared.

Reply:
Agreed, fixed